### DISCUSSION TITLE
Coping with Windows Updates

### DISCUSSION THREAD

**Discussion by SunJii-Khasar** [memstechtips/UnattendedWinstall#130] [General]
I am intereasted in knowing how windows updates affet the changes for this xml file. Ich I install frech and start updatign everything to today's Version, won't that break all the tweaks? How does the file cope with that and what can we do (other thant stop updating immediately) to prevent/correct windows messing with the configuration? Would it be possible to add a section to the README.md and give out until which date the Updates won't mess with the file?

**Comment by memstechtips** [comment]
Unfortunately Windows Updates are known to reinstall bloatware yes, but it shouldn't happen for at least a year. You can always reinstall Windows again once that happens or use a tool like the Chris Titus Windows Utility to remove the bloatware and optimize your PC again.

**Reply by SunJii-Khasar** [reply]
I guess, you think that you could provide any ressources where the community could look up which updates might fall into that category? of course, I don't expect to find out immediately after the release but maybe around a Month later?
